app/assets/javascripts/article-form.js

```diff
@@ -204,6 +206,28 @@ class ArticleForm {
       this.updateCustomUnitWarning();
     });
     this.onSupplierUnitChanged();
+
+    // Add validation for minimum/maximum order quantity
+    this.initializeOrderQuantityValidation();
```

Comment:
Why did you add client side validation to this form? I think since other form fields are validated by the backend, this makes this kind of inconsistent. If I remove that line, it works just as fine - after clicking "Update article", I see the following error:

<img width="590" height="98" alt="image" src="https://github.com/user-attachments/assets/9e44b26f-bb40-44fb-9250-0f9bf48148cb" />

Of course frontend validation is always a nice-to-have in terms of usability, but I was hoping to only add this when building a PWA style client to not further complicate the already ugly jquery code. What do you think?